spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,25 @@
+require 'database_cleaner'
```

Comment:
do we need this file if it's located in `spree_dev_tools` gem ? [source](https://github.com/spree/spree_dev_tools/blob/master/lib/spree_dev_tools/rspec/support/database_cleaner.rb)